pagerdutyplugin/resource_pagerduty_jira_cloud_account_mapping_rule_test.go

```diff
@@ -0,0 +1,247 @@
+package pagerduty
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/PagerDuty/terraform-provider-pagerduty/util"
+	"github.com/hashicorp/terraform-plugin-testing/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+)
+
+func TestAccPagerDutyJiraCloudAccountsMappingRule_Basic(t *testing.T) {
+	accountMappingID := os.Getenv("PAGERDUTY_ACC_JIRA_ACCOUNT_MAPPING_ID")
+	if accountMappingID == "" {
+		t.Skip("Missing env variable PAGERDUTY_ACC_JIRA_ACCOUNT_MAPPING_ID")
+		return
+	}
+
+	rule := fmt.Sprintf("tf-%s", acctest.RandString(5))
+	service := fmt.Sprintf("tf-%s", acctest.RandString(5))
+	user := fmt.Sprintf("tf-%s", acctest.RandString(5))
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { testAccPreCheck(t) },
+		ProtoV5ProviderFactories: testAccProtoV5ProviderFactories(),
+		CheckDestroy:             testAccCheckPagerDutyJiraCloudAccountsMappingRuleDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckPagerDutyJiraCloudAccountsMappingRuleConfig(service, user, rule, accountMappingID),
```

Comment:
What I miss here so far, is an update test case, which could be added for instance to  cover the whole set of PD Priorities, or a new Custom Field for `incident_urgency`. Lastly, a name update is also key to be validated by acc tests.